lib/tests/streamlit/elements/arrow_dataframe_test.py

```diff
@@ -133,3 +135,31 @@ def test_pandas_version_1_3_0_and_above(self, mock_styler_translate):
 
         st._arrow_dataframe(styler)
         mock_styler_translate.assert_called_once_with(False, False)
+
+    @pytest.mark.require_snowflake()
+    def test_snowpark_uncollected(self):
+        """Tests that `styler._translate` is called with correct arguments in Pandas >= 1.3.0"""
+        with create_snowpark_session() as snowpark_session:
+            df = snowpark_session.sql("SELECT 40+2 as COL1")
+            print("===" * 20)
+            print(df)
+            print("===" * 20)
```

Comment:
nit: I'm guessing these are debug logs that should now be removed

Reply:
Removed